addons/payment_adyen/models/payment_transaction.py

```diff
@@ -100,7 +100,8 @@ def _send_refund_request(self, refund_amount=None, create_refund_transaction=Tru
 
         :param float refund_amount: The amount to refund
         :param bool create_refund_transaction: Whether a refund transaction should be created or not
-        :return: None
+        :return: The refund transaction if any, or `None`
+        :rtype: recordset of `payment.transaction`
```

Comment:
In fact, you could simply follow the logic of `_send_payment_request`, `_send_capture_request` and `_send_void_request` and make `_send_refund_request` return None as well since the returned value is not used anywhere :thinking:.